packages/contracts-bedrock/tasks/withdraw-fees.ts

```diff
@@ -0,0 +1,103 @@
+'use strict'
+
+import { ethers } from 'ethers'
+import { task } from 'hardhat/config'
+import * as types from 'hardhat/internal/core/params/argumentTypes'
+import { LedgerSigner } from '@ethersproject/hardware-wallets'
+
+import {
+  constructFeeVaultMulticalls,
+  estimateMulticall,
+  executeMulticalls,
+  multicall3Contract,
+} from '../src/multicall3'
+import { withdrawFeeVault } from '../src/fee-vault'
+
+// Predeployed fee vaults
+// See: https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/contracts/libraries/Predeploys.sol
```

Comment:
You can pull these values from here https://github.com/ethereum-optimism/optimism/blob/6a5cc4ebc24cedc019905f02c0eaa41b3b2bfa7f/packages/contracts-bedrock/src/constants.ts#L9